src/Error.php

```diff
@@ -158,7 +146,7 @@ public function toArray()
      *
      * @return string
      */
-    public function toJson()
+    public function toJson() : string
     {
         return Json::encode($this->toArray());
```

Comment:
This should be confirmed, and also not for that PR: If I remember correctly we are getting rid of Json::encode / Json::decode in favour of requiring ext-json in composer and json_* functions.

About strict types here: `$data` cannot be completely mixed, must be 'serializable'.

Reply:
JSON: Alright, I will open an issue as reminder.

`$data`: I will have to have a closer look to get a better understanding of the context and usage. I will open an issue as reminder.